src/fwupd_ioc.py

```diff
@@ -0,0 +1,146 @@
+# SPDX-License-Identifier: LGPL-2.1+
```

Comment:
in src? contrib perhaps?

Reply:
Anything required for build and test in a "normal" build I don't believe should be in contrib/.